tools/copyblocks/main.go

```diff
@@ -59,6 +60,7 @@ func (c *config) registerFlags(f *flag.FlagSet) {
 	f.Var(&c.enabledUsers, "enabled-users", "If not empty, only blocks for these users are copied.")
 	f.Var(&c.disabledUsers, "disabled-users", "If not empty, blocks for these users are not copied.")
 	f.BoolVar(&c.dryRun, "dry-run", false, "Don't perform any copy; only log what would happen.")
+	f.BoolVar(&c.anyNoCompactBlockDuration, "any-no-compact-block-duration", false, "If set, blocks marked as no-compact are not checked against min-block-duration")
```

Comment:
How about `--skip-no-compact-block-duration-check`?

Reply:
That is way more clear. I'll change it to that, thanks!